capstone/src/main/webapp/script.js

```diff
@@ -31,14 +32,160 @@ async function submitInput() {
   if (!langElement) {
     return;
   }
+
+  // Get the selected attributes
   const attributes = document.getElementById("available-attributes").getElementsByTagName('input');
-  const requestedAttributes = []
+  const requestedAttributes = [];
   for (const attribute of attributes) {
     if (attribute.checked) {
       requestedAttributes.push(attribute.value);	
     }	
   }
-  await callPerspective(textElement.value, langElement.value, requestedAttributes);
+
+  // Get the selected analysis type
+  document.getElementById('analysis-container').innerHTML = '';
+  const radios = document.getElementsByName('analysisRadios');
+  var tokenizer = "";
+  for (i = 0; i < radios.length; i++) {
+    if (radios[i].checked) {
+      tokenizer = radios[i].value;
+      break;
+    }
+  }
+
+  handleInput(textElement.value, langElement.value, requestedAttributes, tokenizer);
+}
+
+/** Submits the input to Perspective and loads the appropriate output */
+async function handleInput(text, lang, requestedAttributes, tokenizer) {
+  // Draw the separating line for the output
+  const separator = document.getElementById('separator-container');
+  separator.innerHTML = '';
+  separator.appendChild(document.createElement('hr')); 
+	
+  // Make Perspective call for the entire submission and load graph data
+  const toxicityData = await callPerspective(text, lang, requestedAttributes);
+  loadChartsApi(toxicityData);
+
+  // Get detailed analysis if requested
+  if (tokenizer != "") {
+    getAnalysis(text, lang, requestedAttributes, tokenizer); 
+  }
+}
+
+/** Prints detailed analysis of text by word or sentence */
```

Comment:
It might be good to add a comment on what the type of `tokenizer` is, or if it's a string what possible values it can take

Reply:
Added in the JSDoc for `TokenizerEnum`